### DISCUSSION TITLE
使用beehive进入答题以后现在似乎会导致小米社区账号被封禁

### DISCUSSION THREAD

**Discussion by Megalounge** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#51] [Show and tell]
本人并没有获得过BootLoader解锁权限，在1月30日使用答题链接进入答题以后，在1月31日显示小米社区被封禁，当天去找小米客服反馈，于2月1日获得电话回访，被告知是因为使用非正当途径进行BL锁解锁答题导致违反用户协议被封禁，无法解封。

我不确定这是不是我个人情况，我看过repo readme里面的警告，明白自己没有通过国内 BootLoader 解锁资格审批才会进入答题链接，不曾想这一行为竟然导致了账号封禁，如果这只是对我的个人影响，那不会有太大影响，反正本人社区等级没有特别高，大不了不答题，如果是最近才开始大规模封禁，那么各位还是得小心为好。

**Comment by xbn2002** [comment]
是大规模，不满5级进入答题全部封禁

**Reply by Megalounge** [reply]
害怕

**Reply by xbn2002** [reply]
你去哪找客服对线的 @Megalounge

**Reply by Megalounge** [reply]
https://www.mi.com/service/contact

**Reply by xbn2002** [reply]
他妈滴，还要排队

**Reply by Megalounge** [reply]
很有可能就是因为大规模封禁导致要排队 我昨天一点都不用排队的

**Reply by xbn2002** [reply]
我这里的反馈是，解锁风控解不了的，理由是账号存在安全风险，客服建议我换号吧。 @Megalounge @chuest

**Reply by YaKun9** [reply]
@xbn2002 封号是只封锁小米社区吗？其他服务会受到影响吗

**Reply by xbn2002** [reply]
@YaKun9 我没被封社区，我是踢出所有内测+bl解锁黑名单风控

**Reply by xbn2002** [reply]
不满5级的答题是直接封社区，具体表现为社区一片白加载不出东西，并且提示“访问小米社区受限，请几天后再试”

**Reply by Megalounge** [reply]
> 不满5级的答题是直接封社区，具体表现为社区一片白加载不出东西，并且提示“访问小米社区受限，请几天后再试”

是的 我这里就是这样的情况

**Reply by xxxx2289** [reply]
![Screenshot_2024-02-04-14-56-41-807_com.xiaomi.vipaccount.jpg](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/139218008/9328b2e2-49aa-4201-bca7-3054c57f7bcd)

5级账户，真服了😂😂就进过一次链接，还是10号

**Reply by xxxx2289** [reply]
> 不满5级的答题是直接封社区，具体表现为社区一片白加载不出东西，并且提示“访问小米社区受限，请几天后再试”

为啥我满了5级也是这样？？？😂

**Comment by xiaojzhao** [comment]
beehive是什么意思啊

**Reply by YaKun9** [reply]
小米答题问卷的域名，也就是通过非社区入口进入答题

**Comment by MlgmXyysd** [comment]
昨天刚加的公告，然后当天就封人，这期之前答题的也被封了，你米真是不知道在干什么

**Reply by xbn2002** [reply]
让你米作呗，伤害老用户的心又不是一天两天了，等他提纯了社区的用户群体再说

**Reply by YaKun9** [reply]
> 让你米作呗，伤害老用户的心又不是一天两天了，等他提纯了社区的用户群体再说

最近答题估计是不会给分了，看到一些等到清零都没给分的，结合最近的动作来看，估计这两周答题也没啥机会了

**Comment by mymine** [comment]
垃圾小米，狗屎小米社区。吃家里老爹的，去舔洋鬼子。

**Reply by xbn2002** [reply]
我社区已经访问受限了，彻底拜拜

**Reply by Megalounge** [reply]
> 我社区已经访问受限了，彻底拜拜

看来大家都一样

**Comment by mymine** [comment]
> 我社区已经访问受限了，彻底拜拜
封号都一两个月了。直接把狗屎社区注销了

**Comment by xxxx2289** [comment]
![Screenshot_2024-02-04-14-56-41-807_com.xiaomi.vipaccount.jpg](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/139218008/987ddf22-f49e-4c60-a3e7-fc54a124a129)

5级账户，真服了😂😂就进过一次链接，还是10号

**Reply by xbn2002** [reply]
社区是寄了，但其他服务应该不受影响

**Reply by xbn2002** [reply]
> 显示这个，我昨天用另一个号去小米社区找社区督导组的他说 用户采取违规方式使用小米帐号，现根据小米账号使用协议，对该账号的小米社区使用权限永久封禁。问他什么违规也不说

OK，那没事了，跟我这里一样

**Reply by xxxx2289** [reply]
> > 显示这个，我昨天用另一个号去小米社区找社区督导组的他说 用户采取违规方式使用小米帐号，现根据小米账号使用协议，对该账号的小米社区使用权限永久封禁。问他什么违规也不说
> 
> OK，那没事了，跟我这里一样

我那个号什么内测都进了，就这样没了真太可惜了，没想到时隔20天还来封我社区，挺让我心寒的😂😂

**Reply by xbn2002** [reply]
> > > 显示这个，我昨天用另一个号去小米社区找社区督导组的他说 用户采取违规方式使用小米帐号，现根据小米账号使用协议，对该账号的小米社区使用权限永久封禁。问他什么违规也不说
> > 
> > 
> > OK，那没事了，跟我这里一样
> 
> 我那个号什么内测都进了，就这样没了真太可惜了，没想到时隔20天还来封我社区，挺让我心寒的😂😂

我也是，我大号隐藏分很高，几乎所有机审的内测有名额都能进，现在也是彻底废了，社区访问不了+解锁风控黑名单（MIUI也解不了）

**Comment by mymine** [comment]
> > 不满5级的答题是直接封社区，具体表现为社区一片白加载不出东西，并且提示“访问小米社区受限，请几天后再试”
> 
> 为啥我满了5级也是这样？？？😂

**Reply by xbn2002** [reply]
现在我也是这样了，有些是满5级，通过申请后答题也直接封社区的
